ui/lib/feed/src/components/reflect-feed.tsx

```diff
@@ -71,15 +68,31 @@ const ReflectFeed: React.FC<ReflectFeedProps> = props => {
     onScrollStateChange,
   } = props;
 
-  //@TODO replace this hook with one handling scroll restoration
-  const reflectionsReq = useInfiniteGetReflectionsFromBeamQuery('last', { id: beamId, last: 10 });
+  //@TODO replace this hook with one handling virtual list
+  const reflectionsReq = useInfiniteGetReflectionsFromBeamQuery(
```

Comment:
this one does not implement any pagination, is this intended?

Reply:
Yes, I had a discussion with Sever to implement virtual list for the beam page. That's why I didn't go about implementing pagination. The code will change.